config/pc/logger-config.h

```diff
@@ -93,7 +93,14 @@
  * Enables current time output at the beginning of log string.
  * Requires #vs_logger_current_time_hal function implementation.
  */
-#define VS_IOT_LOGGER_OUTPUT_TIME   1
+#define VS_IOT_LOGGER_OUTPUT_TIME   0
+
+/** Output current thread title
+ *
+ * Enables current thread title. #VS_IOT_GET_THREAD_ID is used to obtain current thread identifier. Call
+ * #vs_log_thread_descriptor to set human readable thread description. Instead numerical identifier will be output
+ */
+#define VS_IOT_LOGGER_OUTPUT_THREAD_ID 1
```

Comment:
I am not sure here should it be enabled or disabled by default.
From one POV, it is good to resolve some difficult bugs. But in a normal situation this information is exessive:
```
[27/01/2020 14:37:27][DEBUG][MCU1_app_1b:92:da:b4:c4:18]: [INFO]  [main.c:188]  [main thr] Terminating application ...
[27/01/2020 14:37:27][DEBUG][MCU1_app_1b:92:da:b4:c4:18]: [ERROR]  [netif-udp-broadcast.c:96]  [udp bcast thr] UDP broadcast: recv stop
[27/01/2020 14:37:27][DEBUG][MCU1_app_3f:13:9c:23:7f:4a]: [INFO]  [main.c:243]  [netif msg thr] Previous firmware : ver 0.0.0.0, 0
```
So, probably, it would be better to disable it by default, wouldn't be?